README.md

```diff
@@ -91,6 +91,7 @@ KEDA has a number of "scalers" that can both detect if a deployment should be ac
 * Prometheus
 * RabbitMQ
 * Redis Lists
+* Nats Streaming
```

Comment:
Nitpick - Can we order it alphabetically please?

Reply:
Sure thing